internal/admin/dashboard/templates/pagination.html

```
@@ -0,0 +1,9 @@
+{{define "pagination"}}
+<div class="pagination" x-show="{{.}}.total > 0">
+    <span class="pagination-info" x-text="'Showing ' + ({{.}}.offset + 1) + '-' + Math.min({{.}}.offset + {{.}}.limit, {{.}}.total) + ' of ' + {{.}}.total"></span>
+    <div class="pagination-buttons">
+        <button class="pagination-btn" :disabled="{{.}}.offset === 0" @click="{{.}}PrevPage()">Prev</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)